packages/core/src/Stores/pushwoosh-store.js

```diff
@@ -59,7 +59,8 @@ export default class PushwooshStore extends BaseStore {
     sendTags = api => {
         api.getTags()
             .then(result => {
-                if (!result.result['Login ID'] || !result.result['Site Language'] || !result.result.Residence) {
+                // Add Optional chaining to resolve console error Cannot read properties of null (reading 'result')
+                if (!result?.result['Login ID'] || !result?.result['Site Language'] || !result?.result?.Residence) {
```

Comment:
this comment is not related to your PR but have you checked that after this change, your store is filled with data or is null? if it's still null, we need a separate card to fix the issue maybe

Reply:
I agree with you on this